src/core/Object3D.js

```diff
@@ -963,8 +963,8 @@ class Object3D extends EventDispatcher {
 
 }
 
-Object3D.DefaultUp = /*@__PURE__*/ new Vector3( 0, 1, 0 );
-Object3D.DefaultMatrixAutoUpdate = true;
-Object3D.DefaultMatrixWorldAutoUpdate = true;
+Object3D.DEFAULT_UP = /*@__PURE__*/ new Vector3( 0, 1, 0 );
+Object3D.DEFAULT_MATRIXAUTOUPDATE = true;
```

Comment:
What about `DEFAULT_MATRIX_AUTO_UPDATE` and `DEFAULT_MATRIX_WORLD_AUTO_UPDATE`? 

Just want to discuss this now before this comes up at a later point 😇 .

Reply:
I followed the nomenclature suggested by @mrdoob in https://github.com/mrdoob/three.js/pull/25015#discussion_r1041770043.

Reply:
Oh, I did miss that comment 👍 .

Reply:
Unresolving the conversation for now. I'd like to see the response of @mrdoob for https://github.com/mrdoob/three.js/pull/25188#discussion_r1056772931 (since the extended notation was not discussed so far).

Reply:
OK, changed to underscore style.

Reply:
@Mugen87 I modified the code style document ([diff](https://github.com/mrdoob/three.js/wiki/Mr.doob's-Code-Style™/_compare/bea5c6089b4bc530b7b5a90ffd8bc35af84b83e6...5d703514fbabd6400fd856dfde52ab6a18337669)).